proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -419,3 +423,18 @@ message UpdateTshdEventsServerAddressRequest {
   string address = 1;
 }
 message UpdateTshdEventsServerAddressResponse {}
+
+message UpdateHeadlessAuthenticationStateRequest {
+  string cluster_uri = 1;
+  string name = 2;
+  HeadlessAuthenticationState state = 3;
+
+  // HeadlessAuthenticationState is a headless authentication state.
```

Comment:
I'd either remove this comment or add some details to it that are not captured by the enum's name already.

Reply:
Unfortunately I get a lint error if I remove the comment:
```
proto/teleport/lib/teleterm/v1/service.proto:432:3:Enum "HeadlessAuthenticationState" should have a non-empty comment for documentation.
```